man/git-authors.md

```diff
@@ -3,7 +3,9 @@ git-authors(1) -- Generate authors report
 
 ## SYNOPSIS
 
+`git-authors`
 `git-authors` [-l, --list] [--no-email]
+`git-authors` [--output] &lt;filename&gt; [--no-email]
```

Comment:
Thank you for adding docs. You don't need to escape `<` and `>`, and other characters in Markdown as you would in HTML.

Reply:
I follow the document existed and I will try 😄

Reply:
I see, in that case, I would leave it for now. If it's in other places, I think a separate PR should address it. Thanks for letting me know :)